public/resources/js/main.js

```diff
@@ -1,22 +1,22 @@
-document.addEventListener('DOMContentLoaded', function() {
-    var show = document.getElementById('show_cate');
-    var categories = document.querySelector('.categories');
-    categories.addEventListener('click', function() {
-        show.classList.toggle('show_cat');
-        if (show.classList.contains('show_cat')) {
-            categories.querySelector('.more').textContent = 'View Less';
-        } else {
-            categories.querySelector('.more').textContent = 'View More';
-        }
-        localStorage.setItem('categorie_show', show.classList.contains('show_cat') ? 'show' : 'visible');
-    });
+// document.addEventListener('DOMContentLoaded', function() {
+//     var show = document.getElementById('show_cate');
+//     var categories = document.querySelector('.categories');
+//     categories.addEventListener('click', function() {
+//         show.classList.toggle('show_cat');
```

Comment:
delete lines :)

Reply:
do not worry, that is possible script that could be re-used by another team from the frontend team, so i will leave for now, if i see it is not used, then i will delete it